quickjs.go

```diff
@@ -165,6 +165,10 @@ func (r Runtime) RunGC() { C.JS_RunGC(r.ref) }
 
 func (r Runtime) Free() { C.JS_FreeRuntime(r.ref) }
 
+func (r Runtime) SetMemoryLimit(limit uint32) {
```

Comment:
I need to check what Go type is most appropriate here. My thinking so far is that:

* It can not be larger than a C `unsigned long`. I do not remember how an overflow would be handled - but whatever that is it will give someone a bad day.
* It should be the largest type that can be made safe. The calling code should be able to set the limit to the highest value we can safely allow.